### ISSUE TITLE
Working of this code for more than 2 classes

### ISSUE THREAD

**Issue by zBabar** [fchollet/keras#6740]
I have been trying to run this code for reuters dataset but output accuracy is too low. is it a kind of implementation issue or purely linked to fasttext capability for more than 2 classes?

**Comment by kentsommer** [comment]
Without the code that you are trying to run or knowing what accuracy you are expecting it is impossible to help you. 

Additionally please direct these types of implementation/assistance based questions to [stackoverflow](https://stackoverflow.com/questions/tagged/keras)

**Comment by zBabar** [comment]
Thanx for your reply... actually I am talking about an implementation of fasttext in keras...  here is the link for implemenation
https://github.com/fchollet/keras/blob/master/examples/imdb_fasttext.py

I can see in this implementation imdb large dataset has been used as default dataset for testing... it gives almost over 90 accuracy with epoch 5... but when i use dataset with more than two classes it gives accuracy less than 10. 

I am not sure if its kind of implementation issue or....

Because original Fasttext is supposed to work for more than 2 classes as well.... 

I am considering this implementation because its in python.
When i made amendment for more than 2 clasess as given in keras documentation but.....
